stable/README.md

```diff
@@ -105,41 +93,49 @@ Client: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7
-
+You can also verify that the Tiller server is running via `kubectl`.
 ```bash
-oc policy add-role-to-user edit "system:serviceaccount:${TILLER_NAMESPACE}:tiller"
+kubectl get pods -n kube-system
+NAME                                READY   STATUS    RESTARTS   AGE
+...
+tiller-deploy-8c5679674-k9c7m       1/1     Running   0          47m
+...
 ```
 
-The [blog article written by Red Hat][0] suggests having `Tiller` in its own project, separate from application projects. However, this is **NOT** how banks deploy Tiller, it is **NOT** best practice from a security perspective. Banks typically deploy Tiller in each application's project. That way:
 
-- the tiller `edit` role does not have to have `admin` access to the whole cluster
-- each project can be individually managed w.r.t. the version of `tiller` managing its deployments; e.g., no **"big bang"** during an upgrade.
+## Deploying NuoDB using Helm Charts
 
-So instead lets only run our applications alongside `tiller`, in the same project. To deploy a canary application, such as Node.js:
+### Grant OpenShift privileges
 
+First, you should create a new namespace for NuoDB.
 ```bash
-$ helm install https://github.com/jim-minter/nodejs-ex/raw/helm/helm/nodejs-0.1.tgz -n nodejs-ex
+oc new-project nuodb
 ```
 
-Verify that `tiller` properly launched Node.js.
-
+Create a new service account.
 ```bash
-$ oc get pods
-NAME                      READY     STATUS    RESTARTS   AGE
-nodejs-example-1-build    1/1       Running   0          1m
-tiller-86c4495fcc-rhq97   1/1       Running   0          15m
+kubectl -n nuodb create serviceaccount nuodb
 ```
 
-Now that we've seen Helm deploy an application successfully, we're confident in its installation and configuration, lets now move on to deploying NuoDB. But first, lets clean up Node.js:
+Next, you will want to give your new service account the correct SecurityContextConstraints to run NuoDB.
+You can find the recommended SecurityContextConstraints in `deploy/nuodb-scc.yaml`
 
 ```bash
-$ helm del --purge nodejs-ex
+oc apply -f deploy/nuodb-scc.yaml -n nuodb
+oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:nuodb -n nuodb
+oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:default -n nuodb
 ```
 
-## Deploying NuoDB using Helm Charts
+Here is a short list of NuoDB charts and their privilege requirements.
+
+| Charts | Privilege | Short Explanation |
+| ----- | ----------- | ------ |
+| transparent-hugepage| allowHostDirVolumePlugin: true | To mount hostPath and disable THP on host|
+| transparent-hugepage| volumes.hostPath | To mount hostPath and disable THP on host|
+| transparent-hugepage| seLinuxContext.* | To mount hostPath and disable THP on host|
+| admin, database| allowedCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
```

Comment:
According to http://man7.org/linux/man-pages/man7/capabilities.7.html this capability will:
```
Bypass permission checks on operations that normally require
                the filesystem UID of the process to match the UID of the
                file (e.g., chmod(2), utime(2)), excluding those operations
```
My understanding is that we need this only because we execute `chmod 770` in the init containers to prep the volume directories which doesn't fully correspond to what is written in the README.

According to https://stackoverflow.com/questions/43544370/kubernetes-how-to-set-volumemount-user-group-and-file-permissions there are several ways to make sure that the container processes can access files in a volume. Of of the ways is specifying `fsGroup`. It looks like we went down the road with init container which needs the `FOWNER` capability in OCP. I guess that if we decide to change the templates in the charts this will be done in another PR.

Reply:
@Stanimir - this does work, I just checked my notes and I did try a few combinations (setting runAsUser to 0 and fsGroup to 1000 both work).  At the time I moved onto allowing FOWNER in the SCC and setting specifically on the init container as I was trying to limit the changes to where they were needed (and I thought that would be preferable to running as root).  You can't set fsGroup for the init container only and as noted in chat we don't expose the ability to set runAsUser or any security context separately/differently in the init container and container.  We also don't currently have the securityContext.fsGroup logic in the admin chart, only the DB chart.

So this is definitely an option as part of the wider changes in a later PR (Martin raise a Jira already), I'll make sure to add it to the doc too.